pkg/sql/parser/builtins.go

```diff
@@ -246,6 +247,33 @@ var Builtins = map[string][]Builtin{
 		},
 	},
 
+	"from_ip": {
+		Builtin{
+			Types:      ArgTypes{TypeBytes},
+			ReturnType: TypeString,
+			fn: func(_ *EvalContext, args DTuple) (Datum, error) {
+				ipstr := args[0].(*DBytes)
+				nboip := net.IP(*ipstr)
+				sv := nboip.String()
+				if sv == "<nil>" {
+					return DNull, nil
+				}
+				return NewDString(sv), nil
+			},
+		},
+	},
+
+	"to_ip": {
+		Builtin{
+			Types:      ArgTypes{TypeString},
+			ReturnType: TypeBytes,
+			fn: func(_ *EvalContext, args DTuple) (Datum, error) {
+				ipdstr := args[0].(*DString)
+				return NewDBytes(DBytes(net.ParseIP(string(*ipdstr)))), nil
```

Comment:
You need to check the result of ParseIP here; as per the doc it returns nil if the address is invalid, in which case to_ip returns DNull.